src/main/java/com/northconcepts/datapipeline/foundations/examples/decisiontree/AddADecisionTreeToAPipeline.java

```diff
@@ -21,24 +21,28 @@ public static void main(String[] args) {
 
                 .setRootNode(new DecisionTreeNode()
 
-                        .addBranch("${Variant Price} == null || ${Variant Price} < 20", new DecisionTreeNode()
-                            .addOutcome("Shipping", "0.00")
-                            .addOutcome("Total", "${Variant Price} + Shipping")
+                        .addNode(new DecisionTreeNode()
+                                .setCondition("${Variant Price} == null || ${Variant Price} < 20")
```

Comment:
@gvjoshi25 Let's move the condition to the line above instead of putting it on a new line by itself.

Reply:
Done.

Reply:
Let's use the new constructor -- taking a condition -- once you've added it